doc/overview.rst

```diff
@@ -73,11 +72,9 @@ An example is given below. In the example, capacity and mass flow rate are in
 						   "volume": 11 }
 
 The Future
------------
-
-A number of features will be implemented in SaltProc soon. Support of various
-depletion codes (e.g., OpenMC) will be added.
+----------
 
+A number of features will be implemented in SaltProc soon.
```

Comment:
I love this! Maybe we could link to our issues that are tagged as features or wishlist items here too? What do you think?